### ISSUE TITLE
QR-Code not displayed in Google Chrome when center image is defined

### ISSUE THREAD

**Issue by rala72** [Cordobo/angularx-qrcode#206]
I'm facing this issue on Android now.
Windows Chrome Version 110.0.5481.97 is fine; Android 110.0.5481.65 not.

angular: 14.2.12
angularx-qrcode: 14.0.0

Stackblitz: https://angular-kg1kng.stackblitz.io

**Comment by Cordobo** [comment]
Hi @rala72 

thanks for reporting this issue and providing a stackBlitz, I'll look into it.

**Comment by rala72** [comment]
are there any updates on this issue?

**Comment by rala72** [comment]
are there any plans for fixing this?

**Comment by Cordobo** [comment]
@rala72 

yes, it's a shame, it's still open, I cannot reproduce it as I have no android phone which raises this issue. If it is still an issue, a PR would be very welcome.
